src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -198,7 +198,7 @@ Graph QuantizeGraph(Graph &&src) {
         NodePtr mirror_node = mirror_map.at(e.node.get());
         NodeEntry mirror_entry = NodeEntry{
           mirror_node, e.index, e.version};
-        size_t num_outputs = e.node->num_outputs();
+        size_t num_outputs = mirror_node->num_outputs() - 2;
```

Comment:
Yes, I will fix it back for QuantizeGraph pass. Besides, we have added QuantizeGraphUnsigned pass for mkldnn since mkldnn support u8 input data type. So output data type for quantize op will be set u8 in QuantizeGraphUnsigned pass.

Reply:
when mkldnn is enabled, fp32 pooling will have two outputs, one is for workspace, so num_outputs cannot set by fp32 op node when mkldnn enabled.

Reply:
but it looks to me that this change applies to all operators, not just pooling.
also, why 2? if one of the output is workspace, shouldn't it be 1?

Reply:
This need to be reverted and only applies for the new added pass function, I will double check the logic (applied operators and whether should use 1 or 2) and get back to you later.

Reply:
I checked the logic, the reason for this change is the output number (if exclude min/max) of a OP maybe different for FP32 version and INT8 version for CPU, like pooling, for FP32 there may have 2 outputs (one for data one for workspace), but for INT8 version there is only 1 output (data). So here we need to get the quantized OP output number instead of original FP32 OP's. mirror_node is the quantized OP in this case and need to subtract 2 (for min/max) to get min/max's start index. I will add comment here to ease understanding.